pkg/bench/rttanalysis/orm_queries_bench_test.go

```diff
@@ -496,6 +496,50 @@ FROM
       description.objoid = att.attrelid AND description.objsubid = ordinal_position
 WHERE
   table_schema = ANY (ARRAY['public']) AND info.is_hidden = 'NO'
+ORDER BY
+  namespace, table_name, ordinal_position`,
+		},
+		{
+			//See: Prisma: #4250
```

Comment:
nits:
- can you make the comment say what the difference is? (it has the extra where clause)
- can you include the full link to the issue? (actually i don't know what #4250 refers to. doesn't look like https://github.com/prisma/prisma/issues/4250 is related.) 
- maybe also mention why the where clause helps, and mention that if we find a way to fix it on our side we can remove this extra benchmark.

Reply:
Updated to add the differences